wallet/lib/pages/home/home_provider.dart

```
@@ -73,7 +70,7 @@ class HomeProvider extends ChangeNotifier {
       offset: _offset,
     );
     if (response.isLeft()) {
-      LocaleKeys.something_wrong.tr().show();
+      // LocaleKeys.something_wrong.tr().show();
```

Should there be dead code here?

---

Yes i silenced error. Until chain starts receiving custom events.